tests/test_colormap.py

```diff
@@ -1,3 +1,4 @@
+import PySide6
```

Comment:
Since pyqtgraph attempts to be binding-agnostic, we should not import specific bindings outside of rare places like `pyqtgraph/Qt/__init__.py`.

In this case, we already have access to the QtGui module from an import a few lines down.

```python
from pyqtgraph.Qt import QtGui
```

So you can delete the PySide6 import, and in your test code instead of saying `PySide6.QtGui.QColor`, you can say `QtGui.QColor`

Reply:
Thank you, that's good to know! Updated :+1: